src/Streamable.js

```diff
@@ -0,0 +1,17 @@
+export const shouldTransform = url =>
+  /^https?:\/\/streamable\.com\/s\//.test(url);
+
+export const getHTML = url =>
+  `<div
+  height="0"
+  style="position: relative;"
+  width="100%"
+>
+  <iframe
```

Comment:
For consistency with other transformers, I'd like to return only the `iframe` and just put it on 1 line here 🙂

Reply:
In the beginning, I was tempted to do just that, but even through the streamable API, but take a look at even what gets returned through the streamable API:

```javascript
{"status": 2, "files": {"mp4": {"status": 2, "width": 1920, "url": "//cdn-b-east.streamable.com/video/mp4/bx960.mp4?token=sNkeIYq-6hxDBOUl20gpTQ&expires=1576186680", "bitrate": 7992605, "duration": 278.511567, "size": 283068434, "framerate": 30, "height": 1080}, "original": {"framerate": 29.97002997002997, "height": 1080, "width": 1920, "duration": 278.511567, "bitrate": 19968979, "size": 700121965}, "mp4-mobile": {"status": 2, "width": 640, "url": "//cdn-b-east.streamable.com/video/mp4-mobile/bx960.mp4?token=grchii_i30Ds1dizRfSt1A&expires=1576186680", "bitrate": 1187597, "duration": 278.5, "size": 43831942, "framerate": 30, "height": 360}}, "embed_code": "<div style=\"width: 100%; height: 0px; position: relative; padding-bottom: 56.25%;\"><iframe class=\"streamable-embed\" src=\"https://streamable.com/o/bx960\" frameborder=\"0\" scrolling=\"no\" style=\"width: 100%; height: 100%; position: absolute;\" allowfullscreen></iframe></div>", "source": null, "thumbnail_url": "//thumbs-east.streamable.com/image/bx960.jpg?height=300", "url": "streamable.com/bx960", "message": null, "title": "Three Good Things", "percent": 100}
```

Reply:
Hmmm strange 😕

If you just visit https://api.streamable.com/oembed.json?url=https://streamable.com/moo, you don't see that much info tho...

Reply:
Aha - I see what it is. I was curl'ing `https://api.streamable.com/videos/{shortcode}` - the other endpoint they had provided. I'll use the response from your endpoint!

Reply:
Very strange those 2 endpoints give different values for the embed code... 😕

Reply:
Haha, I would agree. But such is the way of the world! I just pushed my changes, hopefully they're up to snuff.